client/blocks/feedback/attributes.js

```diff
@@ -51,7 +51,7 @@ export default {
 	},
 	title: {
 		type: 'string',
-		default: '',
+		default: __( 'Untitled Feedback', 'crowdsignal-forms' ),
```

Comment:
If we give it a default value the condition below where it takes either the title or the header value if no title is set would never work. We probably don't want this.

Also remember to make sure any changes are reflected in the PHP version.

Reply:
Agreed. There was some back and forth about it with Thomas, but I think this will work better. Easy change otherwise